mam/v2/alloc.c

```diff
@@ -0,0 +1,33 @@
+
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * MAM is based on an original implementation & specification by apmi.bsu.by
+ [ITSec Lab]
+
+ *
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+/*!
+\file alloc.c
+\brief MAM2 memory allocator.
+*/
+
+#include "mam/v2/alloc.h"
+
+#include <stdlib.h>
+
+void *mam2_alloc(ialloc *a, size_t s) { return malloc(s); }
```

Comment:
Is it worthy to main MAMv2 specific memory operations?

Reply:
This code is mainly just an import from an external source and will be cleaned up over the course of the next couple days ;)